### ISSUE TITLE
SSO/Standard Login

### ISSUE THREAD

**Issue by mgarf** [wpo365/wpo365-login#14]
This plugin is great, would love to see the ability when navigating to wp-admin to select between regular sign in or SSO rather than having two separate urls.

**Comment by wpo365** [comment]
Hi @mgarf ... What you mean precisely with "having two separate urls"? Do you mean that you can navigate to /wp-login.php and login with a Wordpress-only account?

**Comment by mgarf** [comment]
Yep. So instead of a instant redirect it has the option of WordPress regular sign in or Single Sign On

**Comment by wpo365** [comment]
The premium version of the plugin offers a shortcode that renders a 'Sign in with Microsoft' button. With this shortcode you should be able to develop the customization you wish for. Please consult the online documentation https://www.wpo365.com/authentication-shortcode/
